### PR TITLE
1.9 Development issues, style issues fix

### LINKED ISSUE #3785: showteam page doesn't work with group leaders
This is a v1.9 problem.

The `showteam` page outputs an `Uncaught TypeError: Argument 1 passed to MyBB\Twig\Extensions\UrlExtension::getProfileLink() must be of the type integer, null given, called in...` error when the board has group leaders to show.

Probably caused by neglect during theme migration to Twig. I think I can fix it.

BTW, for the 1.9 development branch, should I wait for the code being rebased before I can pull a request? New to git rebase and current MyBB 1.9 code rebase.

### PR DESCRIPTION
**Pull request by rajat315315**
Resolves #3785

### CODE REVIEW

File: inc/views/base/usercp/attachments.twig [hunk 1]
```diff
@@ -46,7 +46,7 @@
             </li>
             <li class="list__item">
                 <span class="list__field">{{ lang.attachstats_spaceused }}</span>
-                <span class="list__value">{{ friendly_usage }} ({{ percent }})</span>
+                <span class="list__value">{{ friendly_usage }} {{ (percent) }}</span>
```
Comment (euantorano) [inline inc/views/base/usercp/attachments.twig:49]: I'm not sure that this change is correct - I believe the intention was to wrap the output in `()`, but this just wraps the variable and won't be shown in the output.

**Review by euantorano — CHANGES_REQUESTED** [review]
This looks good, thanks. Just a minor comment.

File: inc/views/base/usercp/attachments.twig (hunk 1 above)
Comment (rajat315315) [reply inc/views/base/usercp/attachments.twig:49]: The issue was that when percent is zero, empty brackets were shown.
This change will show `()` only when percent is NOT NULL ??

Comment (euantorano) [reply inc/views/base/usercp/attachments.twig:49]: Ah, ok that makes sense.

**Review by rajat315315 — COMMENTED** [review]
Changes requested resolved.

**Review by euantorano — APPROVED** [review]

**Comment by rajat315315** [comment]
Hello everyone,
Can somebody please nominate me, if I have done some meaningful contribution at the community?
[https://stars.github.com/nominate/](https://stars.github.com/nominate/)

**Pull request merged**